install.sh

```diff
@@ -226,6 +243,7 @@ nvm_do_install() {
     fi
   fi
 
+  nvm_bash_completion
```

Comment:
Okay, would you please give me what is the exactly place I should put this function to? Thanks.